<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DB Manager</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(95, 95, 95);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btnSchema">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(203, 203, 203);</string>
      </property>
      <property name="text">
       <string>Create schema</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnAdd">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(203, 203, 203);</string>
      </property>
      <property name="text">
       <string>Add data</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnShow">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(203, 203, 203);</string>
      </property>
      <property name="text">
       <string>Show data</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>304</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
